rhino/README.md

```diff
@@ -0,0 +1,112 @@
+# Introduction
+
+Rhino supports two editors: IronPython and CPython. The following tutorials explain how to install and use compas on both.
+
+# Installation - Editor: IronPython
+
+- **Step 1:** Install Anaconda: https://www.anaconda.com/products/distribution
+
+- **Step 2:** Anaconda Prompt commands:
+
+    ```bash
```

Comment:
I would copy the other tutorials starting point, which usually just skip ahead the "install conda" part and go straight to creating the environment

Reply:
Removed Anaconda Installation.